internal/storage/inmemory.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+type Storage struct {
+	URLMap map[string]string
+}
+
+func New() *Storage {
+	return &Storage{
+		URLMap: make(map[string]string),
+	}
+}
+
+func (s *Storage) GetValueMap(key string) (string, bool) {
```

Comment:
для тех кто использует пакет совершенно не важно как он реализован внутри - Map тут лишнее. Value тоже не имеет смысла - больше ничего мы получить и не можем кроме значения
Аналогично в set
```suggestion
func (s *Storage) Get(key string) (string, bool) {
```